plugins/serializers/influx/influx.go

```diff
@@ -92,6 +95,19 @@ func (s *Serializer) SetFieldTypeSupport(typeSupport FieldTypeSupport) {
 	s.fieldTypeSupport = typeSupport
 }
 
+func (s *Serializer) SetTimestampUnits(units time.Duration) {
+	unitsNanoseconds := units.Nanoseconds()
+	fmt.Printf("Nanoseconds: %v\n", unitsNanoseconds)
+
+	// if the units passed in were less than or equal to zero,
+	// then serialize the timestamp in seconds (the default)
```

Comment:
I thought the default timestamp format for influx is nanoseconds, not seconds correct?

Reply:
Good catch. Copy paste error from the JSON one which uses seconds by default.